RobotCode/src/main/java/frc/robot/subsystems/ArmSubsytem.java

```diff
@@ -0,0 +1,93 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.MotionMagicDutyCycle;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamTalonFX;
+import frc.robot.RobotContainer;
+import frc.robot.constants.ArmConstants;
+import frc.robot.constants.Ports;
+
+public class ArmSubsytem extends SubsystemBase {
+	private TalonFX armMotorRight, armMotorLeft;
+	private double armLeftMinAngle;
+	public double targetAngle;
+
+	private MotionMagicDutyCycle motorControlRequest;
+
+	public ArmSubsytem()
+	{
+
+		armMotorRight = new TalonFX(Ports.ARM_RIGHT_MOTOR);
+		armMotorLeft = new TalonFX(Ports.ARM_LEFT_MOTOR);
+
+		armMotorRight.setControl(new Follower(armMotorLeft.getDeviceID(), true));
+
+
+		TalonFXConfiguration mainMotorConfigs = new TalonFXConfiguration();
+
+		mainMotorConfigs.CurrentLimits.StatorCurrentLimitEnable = true;
+		mainMotorConfigs.CurrentLimits.StatorCurrentLimit = ArmConstants.ARM_CURRENT_LIMIT;
+
+		// PIDs
+		Slot0Configs pidSlot0Configs = mainMotorConfigs.Slot0;
+		pidSlot0Configs.kP = ArmConstants.ARM_KP;
+		pidSlot0Configs.kI = ArmConstants.ARM_KI;
+		pidSlot0Configs.kD = ArmConstants.ARM_KD;
+		pidSlot0Configs.kS = ArmConstants.ARM_KS;
+		pidSlot0Configs.kV = ArmConstants.ARM_KV;
+		pidSlot0Configs.kA = ArmConstants.ARM_KA;
+		pidSlot0Configs.kG = ArmConstants.ARM_KG;
+
+		armMotorRight.getConfigurator().apply(mainMotorConfigs);
+		armMotorLeft.getConfigurator().apply(mainMotorConfigs);
+
+		targetAngle = 1;
```

Comment:
Set targetAngle equal to a constants value, something like rest position or initial position